presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/PostgreSqlQueryRunner.java

```diff
@@ -58,7 +58,7 @@ public static QueryRunner createPostgreSqlQueryRunner(TestingPostgreSqlServer se
             queryRunner.createCatalog("tpch", "tpch");
 
             Map<String, String> properties = ImmutableMap.<String, String>builder()
-                    .put("connection-url", server.getJdbcUrl())
+                    .put("connection-url", server.getJdbcUrl() + "&stringtype=unspecified")
```

Comment:
It is necessary for CTAS https://github.com/prestosql/presto/pull/81#issuecomment-460539329

Reply:
@ebyhr thanks. This is what I thought. However, I don't think this is the right solution. 
Is this because we use this when writing to json columns?
```
return WriteMapping.sliceMapping("jsonb", varcharWriteFunction());
```

What is the Postgres JDBC API that would work even without `stringtype=unspecified`? 

I'd try `java.sql.PreparedStatement#setObject(int, java.lang.Object)` + `org.postgresql.util.PGobject` with proper type, like https://stackoverflow.com/a/39885334/65458

Reply:
Yes, the PGobject also works. Confirmed below `jsonWriteFunction` passed presto-postgresql's tests.
```java
return WriteMapping.sliceMapping("jsonb", jsonWriteFunction());
...
    private static SliceWriteFunction jsonWriteFunction()
    {
        PGobject pgObject = new PGobject();
        pgObject.setType("json");
        return (statement, index, value) -> {
            pgObject.setValue(value.toStringUtf8());
            statement.setObject(index, pgObject);
        };
    }
```

Reply:
@ebyhr thanks for checking. Nit: move `PGobject pgObject = new PGobject();` inside lambda, don't share nor reuse between threads nor calls.

Reply:
It's needed for the insert of json/jsonb. ```io.prestosql.spi.PrestoException: Batch entry 0 INSERT INTO "tpch"."tpch"."tmp_presto_284aa2d72dd24e5e9e63087446892c35" VALUES ('{}','{"a":1,"b":2}','{"a":[1,2,3],"b":{"aa":11,"bb":[{"a":1,"b":2},{"a":0}]}}','[]') was aborted: ERROR: column "col_0" is of type jsonb but expression is of type character varying```

Reply:
Thanks @ebyhr. I removed ```stringtype=unspecified``` and implemented with ```PGobject``` according to @findepi remarks.